src/main/cpp/multiregion/TravelModel.h

```diff
@@ -101,6 +111,8 @@ class RegionTravel final
 private:
 	RegionId region_id;
 	std::string region_population_path;
+	std::string region_geodistribution_profile_path;
```

Comment:
I see that you've added some fields to `TravelModel`. These don't affect multi-region simulations, though. Right?